Source/core/MessageStore.h

```diff
@@ -25,10 +25,15 @@
 #include "Frame.h"
 
 namespace WPEFramework {
+
 namespace Core {
 
     namespace Messaging {
 
+        extern EXTERNAL const char* MODULE_LOGGING;
+        extern EXTERNAL const char* MODULE_REPORTING;
+        extern EXTERNAL const char* MODULE_INVALID;
```

Comment:
Why are these still needed externally?

Reply:
Just tried it without the EXTERNAL macro and I'm getting such buildroot error:
`[ 85%] Linking CXX executable WPEProcess`
`/home/veith/buildroot/output/host/lib/gcc/arm-buildroot-linux-gnueabihf/9.2.0/../../../../arm-buildroot-linux-gnueabihf/bin/ld: ../messaging/libWPEFrameworkMessaging.so.1.0.0: undefined reference to WPEFramework::Core::Messaging::MODULE_LOGGING`

Reply:
Ok since this is used during control creation as well, let's leave it as it is. Perhaps we will address this another time.